--- authnzerver/tests/test_auth_permissions.py	2022-02-02 14:39:05.977568648 +0000
+++ authnzerver/tests/test_auth_permissions_processedpatch_02380cb7154.py	2022-02-03 11:01:11.434120245 +0000
@@ -21,6 +21,15 @@ def get_test_authdb():
 
 
 def test_role_permissions():
+    user_payload = {'full_name': 'Test User', 'email': 'testuser-passcheck@test.org', 'password': 'aROwQin9L8nNtPTEMLXd', 'pii_salt': 'super-secret-salt', 'reqid': 1}
+    currproc = mp.current_process()
+    if getattr(currproc, 'authdb_engine', None):
+        currproc.authdb_engine.dispose()
+        del currproc.authdb_engine
+    try:
+        os.remove('test-passcheck.authdb.sqlite')
+    except Exception:
+        pass
     '''
     This tests if we can check the permissions for a logged-in user.
 
